tests/helpers/ember.js

```diff
@@ -65,6 +66,7 @@ module.exports = function ember(args, options) {
   }
 
   cliInstance = cli({
+    process: new MockProcess(),
     inputStream,
```

Comment:
Seems like we could pass `inputStream` and `outputStream` from the `MockProcess` instance.
Not sure right now which benefits does it offer but it seems more consistent.